CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.13)
+
+# Configuration options
+option(vcdiff_build_tests "Build all of vcdiff tests." ON)
+option(vcdiff_build_exec "Build vcdiff exectuable" ON)
+
+# Add subprojects first so we won't interfere with config.h.in
+
+IF (vcdiff_build_exec)
+  SET(GFLAGS_BUILD_gflags_LIB OFF)
+  SET(GFLAGS_BUILD_gflags_nothreads_LIB ON)
+  SET(GFLAGS_INSTALL_HEADERS OFF)
+  SET(GFLAGS_INSTALL_SHARED_LIBS OFF)
+  SET(GFLAGS_INSTALL_STATIC_LIBS OFF)
+  ADD_SUBDIRECTORY(gflags)
+ENDIF()
+
+IF (vcdiff_build_tests)
+  SET(gtest_install OFF CACHE BOOL "Don't install gtest" FORCE)
+  ADD_SUBDIRECTORY(googletest/googletest)
+ENDIF()
+
+PROJECT(open-vcdiff CXX C)
+SET(PROJECT_VERSION 1.0)
+SET(PROJECT_SOVERSION 1)
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -O2")
```

Comment:
These should not be set here. CMake should automagically generate the right flags depending on the config you select. I think you were trying to make a "release" build with debug info, correct? This should work with the configuration "RelWithDebInfo". Please remove this line.